src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
```

Comment:
You need to fix linter errors
<img width="946" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/64750958/c1a34638-0578-449d-b904-d5d219210712">

Reply:
<img width="1068" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/64750958/4ec16959-c62a-4d1c-92eb-27200b1c6333">